app/javascript/components/widgets/twitter/tweet_profile.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import twitterIcon from '@images/twitter-mojo.svg';
+import { fontSizes, colors, spacing } from '@lib/theme';
+
+const ProfileWrapper = styled.div`
+  display: flex;
+  flex-wrap: no-wrap;
+  justify-content: flex-start;
+  width: 18vw;
+  flex-direction: row;
+  align-items: flex-end;
+  margin-bottom: 48px;
+`;
+
+const ProfileInfoWrapper = styled.div`
+  display: flex;
+  flex-wrap: no-wrap;
+  justify-content: space-between;
+  flex-direction: column;
+`;
+
+const ProfileHeader = styled.div`
+  color: ${colors.white}
+  font-size: ${fontSizes.large}
+  margin-bottom: ${spacing.s}
+`;
+
+const ProfileSub = styled.div`
+  color: ${colors.white}
+  opacity: 0.5
+  font-size: ${fontSizes.small}
+`;
+
+const ProfileIcon = styled.img`
+  margin-right: 24px;
+`;
+
+const TwitterProfile = ({ dateCreated }) => {
+  return (
+    <ProfileWrapper>
+      <ProfileIcon src={twitterIcon} alt="twitter profile" />
+      <ProfileInfoWrapper>
+        <ProfileHeader>MojoTech</ProfileHeader>
+        <div style={{ display: 'flex', flexDirection: 'row' }}>
+          <ProfileSub>@MojoTech</ProfileSub>
+          <ProfileSub style={{ marginLeft: '15px', marginRight: '15px' }}>
```

Comment:
Avoid inline styles here and above with styled component. Could the spaces around `•` be replaced with left/right padding?